homeassistant/components/pterodactyl/sensor.py

```diff
@@ -0,0 +1,195 @@
+        value_fn=lambda data: data.memory_limit,
+        device_class=SensorDeviceClass.DATA_SIZE,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        native_unit_of_measurement=UnitOfInformation.MEGABYTES,
+        suggested_unit_of_measurement=UnitOfInformation.GIGABYTES,
+        suggested_display_precision=1,
+        entity_registry_enabled_default=False,
+    ),
+    PterodactylSensorEntityDescription(
+        key=KEY_DISK_USAGE,
+        translation_key=KEY_DISK_USAGE,
+        value_fn=lambda data: data.disk_usage,
+        device_class=SensorDeviceClass.DATA_SIZE,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        native_unit_of_measurement=UnitOfInformation.BYTES,
+        suggested_unit_of_measurement=UnitOfInformation.GIGABYTES,
+        suggested_display_precision=1,
+    ),
+    PterodactylSensorEntityDescription(
+        key=KEY_DISK_LIMIT,
+        translation_key=KEY_DISK_LIMIT,
+        value_fn=lambda data: data.disk_limit,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        device_class=SensorDeviceClass.DATA_SIZE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfInformation.MEGABYTES,
+        suggested_unit_of_measurement=UnitOfInformation.GIGABYTES,
+        suggested_display_precision=1,
+        entity_registry_enabled_default=False,
+    ),
+    PterodactylSensorEntityDescription(
+        key=KEY_NETWORK_INBOUND,
+        translation_key=KEY_NETWORK_INBOUND,
+        value_fn=lambda data: data.network_inbound,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        device_class=SensorDeviceClass.DATA_SIZE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfInformation.BYTES,
+        suggested_unit_of_measurement=UnitOfInformation.GIGABYTES,
+        suggested_display_precision=1,
+        entity_registry_enabled_default=False,
+    ),
+    PterodactylSensorEntityDescription(
+        key=KEY_NETWORK_OUTBOUND,
+        translation_key=KEY_NETWORK_OUTBOUND,
+        value_fn=lambda data: data.network_outbound,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        device_class=SensorDeviceClass.DATA_SIZE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfInformation.BYTES,
+        suggested_unit_of_measurement=UnitOfInformation.GIGABYTES,
+        suggested_display_precision=1,
+        entity_registry_enabled_default=False,
+    ),
+    PterodactylSensorEntityDescription(
+        key=KEY_UPTIME,
+        translation_key=KEY_UPTIME,
+        value_fn=lambda data: data.uptime,
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTime.MILLISECONDS,
+        suggested_unit_of_measurement=UnitOfTime.DAYS,
+        suggested_display_precision=1,
+    ),
```

Comment:
Ideally we make this a timestamp sensor, as that one writes state way less and only on change

Reply:
I pushed it like that. Let me know if it can be done better.

Reply:
please wrap the lambda in parenthesis as they now span multiple lines

Reply:
Done 👍